docs/book/src/cronjob-tutorial/testdata/emptymain.go

```diff
@@ -97,3 +98,26 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+/*
+
+Note that the Manager can restrict the namespace that all controllers will watch for resources by:
+
+```go
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	Scheme: scheme,
+	Namespace: namespace,
+	MetricsBindAddress: metricsAddr}
+)
+```
+
+The above example will change the scope of your project to a Namespaced one and then, in this scenario
+would also be suggested replace the default ClusterRole and ClusterRoleBinding to Role and RoleBinding respectively.
+For further information see the kubernetes documentation about [Using RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+
+<aside class="note">
+<h1>Watching a list of Namespaces</h1>
```

Comment:
IMO it's unnecessary to add a note class here. And it seems markdown syntax doesn't work in this block.